internal/config/context.go

```diff
@@ -94,6 +94,47 @@ func CurrentContext() (endpoint Endpoint, user *User) {
 	return
 }
 
+// GetCurrentContextInfo returns endpoint and basic auth info
+// associated with current context
+func GetCurrentContextInfo() (server string, username string, password string) {
+	endpoint, user := CurrentContext()
+	server = fmt.Sprintf("%s,%d", endpoint.Address, endpoint.Port)
+	username = user.BasicAuth.Username
+	if user.AuthenticationType == "basic" {
+		password = decryptCallback(
+			user.BasicAuth.Password,
+			user.BasicAuth.PasswordEncrypted,
+		)
+	}
+	return
+}
+
+// isCurrentContextValid returns true if it has valid information associated
```

Comment:
I (captial)

Reply:
Removed this function as it was not needed.